src/main/java/liquibase/ext/cassandra/database/CassandraDatabase.java

```diff
@@ -1,163 +1,166 @@
-		}
-		return null;
-	}
-
-	@Override
-	public boolean supportsTablespaces() {
-		return false;
-	}
-
-	@Override
-	public boolean supportsRestrictForeignKeys() {
-		return false;
-	}
-
-	@Override
-	public boolean supportsDropTableCascadeConstraints() {
-		return false;
-	}
-
-	@Override
-	public boolean isAutoCommit(){
-		return true;
-	}
-
-	@Override
-	public void setAutoCommit(boolean b){
-	}
-
-	@Override
-	public boolean isCaseSensitive() {
-		return true;
-	}
-
-	@Override
-	public String getCurrentDateTimeFunction() {
-		// no alternative in cassandra, using client time
-		return String.valueOf(System.currentTimeMillis());
-	}
-
-	public String getKeyspace() {
-		if (keyspace == null) {
-			try {
-				if (this.getConnection() instanceof JdbcConnection) {
-					keyspace = ((CDBJDBCConnection) ((S42Connection) ((JdbcConnection) (this).getConnection())
-							.getUnderlyingConnection()).getConnection()).getSession().getLoggedKeyspace();
-				}
-			} catch (Exception e) {
-				Scope.getCurrentScope().getLog(CassandraDatabase.class)
-						.severe("Could not get keyspace from connection", e);
-
-			}
-		}
-		return keyspace;
-
-	}
-
-	@Override
-	public boolean supportsSchemas() {
-		return false;
-	}
-
-	/**
-	 * Cassandra actually doesn't support neither catalogs nor schemas, but keyspaces.
-	 * As default liquibase classes don't know what is keyspace we gonna use keyspace instead of catalog
-	 */
-	@Override
-	public String getDefaultCatalogName() {
-		return getKeyspace();
-	}
-
-	public Statement getStatement() throws DatabaseException {
-		return ((JdbcConnection) super.getConnection()).createStatement();
-	}
-
-	@Override
-	public boolean jdbcCallsCatalogsSchemas() {
-		return true;
-	}
-
-	@Override
-	public boolean supportsNotNullConstraintNames() {
-		return false;
-	}
-
-	@Override
-	public boolean supportsPrimaryKeyNames() {
-		return false;
-	}
-
-   /**
-   * there shouldn't be keyspace name before the index name, queries fail otherwise
-   */
-	@Override
-	public String escapeIndexName(String catalogName, String schemaName, String indexName) {
-		return this.escapeObjectName(indexName, Index.class);
-	}
-}
+package liquibase.ext.cassandra.database;
+
+import com.simba.cassandra.cassandra.core.CDBJDBCConnection;
+import com.simba.cassandra.jdbc.jdbc42.S42Connection;
+import liquibase.Scope;
+import liquibase.database.AbstractJdbcDatabase;
+import liquibase.database.DatabaseConnection;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.DatabaseException;
+import liquibase.structure.core.Index;
+
+import java.sql.Statement;
+
+/**
+ * Cassandra 1.2.0 NoSQL database support.
+ */
+public class CassandraDatabase extends AbstractJdbcDatabase {
+	public static final String PRODUCT_NAME = "Cassandra";
+	public static final String SHORT_PRODUCT_NAME = "cassandra";
+	public static final Integer DEFAULT_PORT = 9160;
+	public static final String DEFULT_DRIVER = "com.simba.cassandra.jdbc.Driver";

```

Comment:
Hi, is there a specific reason this was changed from the previous value of `com.simba.cassandra.jdbc42.Driver`?

Reply:
Good catch. Is it causing problems now?

Reply:
We were using the previous default driver (SimbaCassandraJDBC42) on our project, hence when we updated to the latest version we started getting `Unexpected error running Liquibase: Cannot find database driver: com.simba.cassandra.jdbc.Driver` errors.

Was this default driver update intended or just a cleanup hiccup?

Reply:
I believe it was a cleanup hiccup . Could you open the PR for the fix? Then we can merge it and release 4.23.1.1 .

Reply:
@filipelautert I opened https://github.com/liquibase/liquibase-cassandra/pull/206, please review it when you have a minute.

Thanks.